### PR TITLE
CRM-19936 Added unit test to verify line item count for multiple memb…

### PR DESCRIPTION
**Pull request by Edzelopez**
…ership priceset

----------------------------------------
* CRM-19936: Membership line items are duplicated when a membership is created or renewed with priceset containing multiple memberships
  https://issues.civicrm.org/jira/browse/CRM-19936

### CODE REVIEW

**Comment by Edzelopez** [comment]
This extends functionality patched at PR#9744

File: tests/phpunit/CiviTest/CiviUnitTestCase.php [hunk 1]
```diff
@@ -3767,4 +3767,46 @@ public function createPriceSetWithPage($entity = NULL, $params = array()) {
     $this->_ids['membership_type'] = $membershipTypeID;
   }
 
+  public function setupMultipleMembershipPriceset($number = 1) {
```
Comment (eileenmcnaughton) [inline tests/phpunit/CiviTest/CiviUnitTestCase.php]: Just a note that I think we are going to have to move to have comment blocks for functions in the test suite at some point - not a must-fix-now - but something to try to do next time.

If we run phpcs over the code & want to raise the bar on our standards I don't think we can have higher standards on some directories than others - I think Tim tried.

**Comment by eileenmcnaughton** [comment]
@Edzelopez this is stale but I approve it if you can fix that - one the basis that more tests = more tests

**Comment by monishdeb** [comment]
@Edzelopez can you rebase the PR as it gone stale.

**Comment by Edzelopez** [comment]
@eileenmcnaughton @monishdeb Sorry for the delay on this. I have rebased and added the docblock.

**Comment by eileenmcnaughton** [comment]
It's failing on an enotice
```
api_v3_ContributionPageTest::testCreateMultipleMemberships
Failure in api call for contribution_page submit:  Undefined index: tax_amount
#0 /home/jenkins/buildkit/build/core-9756-7jbgk/sites/all/modules/civicrm/CRM/Contribute/Form/Contribution/Confirm.php(983): PHPUnit_Util_ErrorHandler::handleError(8, 'Undefined index...', '/home/jenkins/b...', 983, Array)
```

**Pull request merged**